src/events/message.rs

```diff
@@ -13,6 +16,22 @@ pub async fn message(ctx: client::Context, msg: Message) -> Result<()> {
         return Ok(());
     }
 
+    if let Some(image) = msg
+        .attachments
+        .iter()
+        .filter(|a| a.height.is_some() && a.height.unwrap() > 0)
+        .next()
+    {
```

Comment:
Same as below regarding `.height.is_some() && a.height.unrwap() > 0` vs just `.dimensions().is_some()`,
Additionally, you can use `.find` as a cleaner way to do `.filter(...).next()`